requirements/edx/github.in

```diff
@@ -106,3 +106,5 @@
 -e git+https://github.com/mitodl/edx-sga.git@6b2f7aa2a18206023c8407e2c46f86d4b4c3ac96#egg=edx-sga==0.8.0
 -e git+https://github.com/open-craft/xblock-poll@7ba819b968fe8faddb78bb22e1fe7637005eb414#egg=xblock-poll==1.2.7
 -e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@2.1.5#egg=xblock-drag-and-drop-v2==2.1.5
+-e git+https://github.com/mitodl/rapid-response-xblock.git@4251bb15124bdf0b681b431fa1cd67fd094387c4#egg=rapid-response==0.1.0
```

Comment:
Do we need to run the pip-compile as well or does that get executed as part of the Ansible build?

Reply:
I was going off of the [README](https://github.com/mitodl/rapid-response-xblock) file here and it there is no mention of `pip-compile`